webpack.config.js

```diff
@@ -0,0 +1,20 @@
+const path = require('path');
+
+module.exports = {
+  entry: './client/src/index.js',
+  output: {
+    path: path.join(__dirname, '/client/public'),
+    filename: 'main_bundle.js'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(js?|jsx?)/,
+        loader: 'babel-loader',
+        options: {
+          'presets': ['@babel/preset-env', '@babel/preset-react']
```

Comment:
I'll have to revisit the requirements, but I vaguely recall we are allowed to use ES6 without transpiling, so you may not actually need preset-env. No harm in doing so though.

Reply:
I thought I ran into some bugs before. I will leave it for now, and get back to it later.